packages/locale/res/translations/_template.json

```diff
@@ -451,6 +451,46 @@
     "This group does not have access to any vaults yet.",
```

Comment:
Can we agree to wait with updating the translations until right before merging? Makes the PR a little less noisy.

Reply:
Ok, I just like to see the tests passing, but can live with that.